benches/gzip.rs

```diff
@@ -71,7 +71,10 @@ fn serve(b: &mut test::Bencher, level: u32) {
     // Should be plenty for effective or ineffective compression.
     let mut buf = Vec::with_capacity(WONDERLAND.len());
     let mut run = || {
-        let mut resp = client.get(&format!("{}/{}", &*SERVER, level)).send().unwrap();
```

Comment:
(I just pushed a standalone rustfmt run to eliminate these diffs.)

Reply:
Okay, will rebase